src/engine/enginemaster.cpp

```diff
@@ -664,11 +664,12 @@ void EngineMaster::process(const int iBufferSize) {
             }
         }
 
-        // Submit buffer to the side chain to do broadcasting, recording,
-        // etc. (CPU intensive non-realtime tasks)
-        // If recording/broadcasting from a sound card input,
-        // SoundManager will send the input buffer from the sound card to m_pSidechain
-        // so skip sending a buffer to m_pSidechain here.
+        // Submit buffer to the side chain to do CPU intensive non-realtime
+        // tasks like recording. The SoundDeviceNetwork, responsible for
+        // passing samples to the network reads directly from m_pSidechainMix.
+        // If recording/broadcasting from a sound card input.
+        // Note: In case the broadcast/recording input is configured,
+        // SoundManager has copied the input buffer to m_pSidechainMix before.
```

Comment:
Seems the "If recording/broadcasting from a sound card input." line is not needed now.
Also, it might be confusing to say that SoundManager has copied the input buffer. The copy happens on EngineSideChain, when called from SoundDeviceNetwork when called from SoundManager.